$NetBSD: patch-ab,v 1.6 2018/01/10 10:10:10 wiz Exp $

Disable ucontext.h detection, which leads to problems on some platforms.
Disable univeral binary build on Mac OS X.

--- configure.orig	2017-11-17 23:05:28.000000000 +0000
+++ configure
@@ -15851,7 +15851,7 @@ $as_echo "#define STDC_HEADERS 1" >>conf
 
 fi
 
-for ac_header in limits.h unistd.h ucontext.h
+for ac_header in limits.h unistd.h
 do :
   as_ac_Header=`$as_echo "ac_cv_header_$ac_header" | $as_tr_sh`
 ac_fn_c_check_header_mongrel "$LINENO" "$ac_header" "$as_ac_Header" "$ac_includes_default"
@@ -16172,7 +16172,7 @@ LIB_SUFFIX=""
 lib_suffix_valid="no"
 
 lib_namespace=""
-if test "x${library_namespace_versioning}" == xyes ; then
+if test "x${library_namespace_versioning}" = xyes ; then
     cat >>confdefs.h <<_ACEOF
 #define IMATH_INTERNAL_NAMESPACE Imath_${ILMBASE_VERSION_API}
 _ACEOF
@@ -16191,7 +16191,7 @@ _ACEOF
     lib_namespace="${ILMBASE_VERSION_API}"
     LIB_SUFFIX="${ILMBASE_VERSION_API}"
     lib_suffix_valid="yes"
-elif test "x${library_namespace_versioning}" == xno ; then
+elif test "x${library_namespace_versioning}" = xno ; then
     cat >>confdefs.h <<_ACEOF
 #define IMATH_INTERNAL_NAMESPACE Imath
 _ACEOF
@@ -16227,7 +16227,7 @@ _ACEOF
 fi
 
 
-if test "x${lib_suffix_valid}" == xyes ; then
+if test "x${lib_suffix_valid}" = xyes ; then
 LIB_SUFFIX_DASH="-${LIB_SUFFIX}"
 
  if true; then
@@ -16262,7 +16262,7 @@ else
 fi
 
 
-if test "x${custom_public_namespace}" == xyes ; then
+if test "x${custom_public_namespace}" = xyes ; then
     { $as_echo "$as_me:${as_lineno-$LINENO}: WARNING: Enabling 'custom user namespace' requires an additional argument, reverting to default" >&5
 $as_echo "$as_me: WARNING: Enabling 'custom user namespace' requires an additional argument, reverting to default" >&2;}
     $as_echo "#define IMATH_NAMESPACE Imath" >>confdefs.h
@@ -16274,7 +16274,7 @@ $as_echo "$as_me: WARNING: Enabling 'cus
 
     public_namespace=""
     custom_public_namespace=no
-elif test "x${custom_public_namespace}" == xno ; then
+elif test "x${custom_public_namespace}" = xno ; then
     $as_echo "#define IMATH_NAMESPACE Imath" >>confdefs.h
 
     $as_echo "#define IEX_NAMESPACE Iex" >>confdefs.h
@@ -16315,7 +16315,7 @@ else
 fi
 
 
-  if test "${osx_arch}" != default ; then
+  if false ; then
           osx_arch_flags="-arch ${osx_arch// / -arch }"
      CXXFLAGS="$CXXFLAGS $osx_arch_flags"
 
